cmd/state-svc/main.go

```diff
@@ -50,19 +50,24 @@ func main() {
 
 	cfg, err := config.New()
 	if err != nil {
-		// We do not want to log an error here as we want to avoid potential rollbar reports until we load the config
-		logging.Debug("Failed to load configuration: %v", err)
-	} else {
-		logging.CurrentHandler().SetConfig(cfg)
+		logging.Critical("Could not initialize config: %v", errs.JoinMessage(err))
```

Comment:
We should print something to the user also, otherwise it just exits with code `1` and no explanation.